docs/source/packs.rst

```diff
@@ -123,6 +123,9 @@ Finally, you can install a pack from existing local dir:
     # Install a pack from '/home/stanley/bitcoin' dir
     st2 pack install file:///home/stanley/bitcoin
 
+.. warning::
+
+    Installing a pack from a directory that is a git repository will only install the latest commit, and ignores any subsequent changes to the files.
```

Comment:
Perhaps we should make it more explicit and say something like ``and ignores any subsequent uncommitted changes to the files.``?

Since ``ignores any subsequent changes to the files`` by itself is a bit confusing.